velox/exec/Task.cpp

```diff
@@ -1432,6 +1435,13 @@ TaskStats Task::taskStats() const {
           .operatorStats[statsCopy.operatorId]
           .add(statsCopy);
     }
+    if (driver->isOnThread()) {
+      ++taskStats.runningDrivers;
+    } else if (driver->isTerminated()) {
+      ++taskStats.terminatedDrivers;
+    } else {
+      ++taskStats.blockedDrivers[(int)driver->blockingReason()];
```

Comment:
you can define an unordered map for this?
```
++taskStats.numBlockedDriversByReason[driver->blockingReason()];

std::unordered_map<BlockingReason, int32_t> numBlockedDriversByReason;
```

Reply:
Why? It is a short enum.
std::array is perfect for this.